.github/workflows/integration.yml

```diff
@@ -34,10 +36,26 @@ on:
   # manual trigger
   workflow_dispatch:
     inputs:
-      dbt-core-branch:
+      dbt_core_branch:
         description: "branch of dbt-core to use in dev-requirements.txt"
         required: false
         type: string
+        default: main
+      name:
```

Comment:
What's the value of this parameter. Is it really just to insert into `run-name` to identify the dispatched run? I feel like `github.event_name` and `github.actor` would suffice. Removing this would simplify the interface.

Reply:
I want to be able to query all of the runs that I've created for a particular pull request, and event_name and actor are not enough to determine that. If github's apis returned the run id like it should this would be moot, but it doesn't. The only way that I found to find the run that you dispatched is by matching on a unique name.  Right now I'm working on FOUR different pull requests that need this testing and if all I have is event_name and actor they'll all get mixed up.